README.md

```diff
@@ -52,3 +52,54 @@ cmake -DLIBOBS_INCLUDE_DIR="../../obs-studio/libobs" -DCMAKE_INSTALL_PREFIX=/usr
 make -j4
 sudo make install
 ```
+
+## Build [RHEL/CentOS]
+
+#### Dependencies
+
+If you are compiling this on Fedora, I would suggest installing the following dependencies inside a `toolbox` ([here](https://fedoramagazine.org/a-quick-introduction-to-toolbox-on-fedora/) you can learn more about it)
+
+- QT Development, the package has the following name:
+    - `qt5-qtbase-devel`
+
+- `Development Tools` group (install with `sudo yum groupinstall "Development Tools"`)
+
+- The following packages are not included in the Development Tools group, but are required:
+    - `cmake`
+    - `g++`
+
+- RPM Fusion Free ([here](https://rpmfusion.org/Configuration) you can learn how to enable this repository)
+    - `obs-studio-libs`
```

Comment:
Uneeded (implementation detail), this will be fetch automatically once obs-studio-devel is installed.